handler/openid/flow_implicit_test.go

```diff
@@ -241,9 +241,24 @@ func TestImplicit_HandleAuthorizeEndpointRequest(t *testing.T) {
 				return makeOpenIDConnectImplicitHandler(4)
 			},
 			check: func() {
-				assert.NotEmpty(t, aresp.GetFragment().Get("id_token"))
-				assert.NotEmpty(t, aresp.GetFragment().Get("state"))
-				assert.NotEmpty(t, aresp.GetFragment().Get("access_token"))
+				assert.NotEmpty(t, aresp.GetParameters().Get("id_token"))
+				assert.NotEmpty(t, aresp.GetParameters().Get("state"))
+				assert.NotEmpty(t, aresp.GetParameters().Get("access_token"))
+			},
+		},
+		{
+			description: "default responseMode check",
```

Comment:
Same as for the others, check for other response modes with expected success or errors.

Reply:
+1 but it doesn't make sense to check success/error for other modes because the validation happens in `authorize_response_writer`. I have added a test case there.

Reply:
Also added integration test to cover a various combination of response types and modes